cli/tests/testdata/bench/no_check.out

```diff
@@ -1,5 +1,9 @@
-error: Uncaught TypeError: Cannot read properties of undefined (reading 'fn')
+error: TypeError: Cannot read properties of undefined (reading 'fn')
```

Comment:
These messages may change as we continue refactoring -- it's a bit annoying that they change because of these refactorings but I'll try to stabilize them with the future work.